spec/factories/journey.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :journey do
+    annual_miles { 365 }
+    monthly_miles { 30 }
+    weekly_miles { 7 }
```

Comment:
These are hundredths of a mile now, right? These seem like very conservative goals 🧌

Reply:
Yeah I didn't really think that one through very well. For now I was just putting numbers there to get my tests to pass. Going forward we will be using these same factories for more in depth tests where it would probably help if the numbers were realistic, right?

Reply:
It will but you can always pass new ones to the factories. This isn't a blocker, in my opinion, just a comment.